doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,502 @@
+  * Added two new allowed offsets for the struct: 0x0 and 0x1000 bytes.
+  * Removed ``memeq()`` in favor of regular ``memcmp()``.
+  * Renamed ``__ext_api()`` to ``EXT_API()``, because names starting with ``__`` are reserved for the compiler.
+  * Added new configuration options ``CONFIG_*_EXT_API_REQUIRED`` and ``CONFIG_*_EXT_API_ENABLED`` for, respectively, users and providers of EXT_APIs.
+
+* :ref:`lib_dfu_target`:
+
+  * Added the configuration option :option:`CONFIG_DFU_TARGET_MCUBOOT_SAVE_PROGRESS`, which uses Zephyr's :ref:`zephyr:settings` subsystem.
+    When this option is enabled, the write progress of an MCUboot style upgrade is stored, so that the progress is retained when the device reboots.
+  * Fixed a bug where :cpp:func:`dfu_target_done` printed ``unable to deinitialize dfu resource`` when no target was initialized. ??? did it actually print it or return an error?
+
+* Moved the following libraries from ``drivers/`` to ``lib/``:
+
+  * :ref:`fprotect_readme`
+  * :ref:`st25r3911b_nfc_readme`
+  * ``adp536x``
+  * ``flash_patch``
+
+
+Crypto
+======
+
+* Added low-level cryptographic test suite using NIST, RFCs, and custom test vectors.
+* :ref:`nrf_cc310_mbedcrypto_readme`/:ref:`nrf_cc310_platform_readme` v0.9.2:
+
+  * Fixed power-efficiency issues.
+  * Added experimental use of CryptoCell interrupt instead of busy-waits.
+* :ref:`lib_hw_cc310`:
+
+  * Added support for CryptoCell interrupt.
+
+nRF BLE Controller
+==================
+
+* Updated the :ref:`nrfxlib:ble_controller` to v. ??? what version? The changelog seems to be messed up.
+  For details, see the :ref:`nrfxlib:ble_controller_changelog`. ??? update link
+
+Multi-Protocol Service Layer (MPSL)
+===================================
+
+* Added MPSL v0.5.0-1.prealpha, a library of common services for single and multi-protocol implementations.
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+???
+
+
+NFC
+---
+
+???
+
+nRF Desktop
+===========
+
+* Added a ``ble_qos`` module to maintain channel maps.
+
+Build system
+============
+
+* Fixed a bug where a user-defined HEX file was not included in the merge operation. ??? what is the context here? Partition Manager?
+
+nrfx
+====
+
+* Updated to v2.1.0.
+  For details, see the `changelog <https://github.com/NordicSemiconductor/nrfx/blob/v2.1.0/CHANGELOG.md>`_.
+
+
+??? Where does this go?
+=======================
+
+Settings: Reduced the default partition size for settings storage.
+
+
+Documentation
+=============
+
+* Added or updated documentation for the following samples:
+
+  * nRF9160:
+
+    * :ref:`cloud_client` - added
+    * :ref:`gps_with_supl_support_sample` - added
+    * :ref:`https_client` - added
+    * :ref:`serial_lte_modem` - added
+    * :ref:`nrf_coap_client_sample` - updated
+
+  * Bluetooth Low Energy:
+
+    * :ref:`ble_llpm` - added ??? not mentioned in release notes
+    * :ref:`bluetooth_mesh_light` - added ??? not mentioned in release notes
```

Comment:
@trond-snekvik, please cover missing parts for Bluetooth mesh

Reply:
We decided to leave Mesh out of the release notes, so no input needed after all.
I'll remove the mentions here.